modules/roxctl-image-command-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cli/checking-policy-compliance.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="roxctl-image-command-overview_{context}"]
+= roxctl image command overview
+
+Commands that you can run on a specific image.
+
+.Usage
+[source,terminal]
+----
+$ roxctl image [command] [flags]
+----
+
+.Available commands
+[cols="2,2",options="header"]
+|===
+|Command |Description
+
+|`check`       
+|Check images for build time policy violations, and report them.
+  
+|`scan`        
+|Scan the specified image, and return the scan results.
+|===
+
+.Options
+[cols="2,2",options="header"]
+|===
+
+|`-t`, `--timeout duration`
+|Set the timeout for API requests representing the maximum duration of a request. The default value is `10m0s`.
```

Comment:
This is showing up bold in the preview. I think because there isn't a Command - Description line. So I'd either add that line, or remove the header option from the table description so the table doesn't think that this is a header.